README.md

```diff
@@ -126,25 +126,15 @@ pip install -e .
 
 ### Linux
 
-First, you need to install protobuf.
-
-Ubuntu users: the quickest way to install protobuf is to run
+First, you need to install protobuf. The minimum Protobuf version required by ONNX is 3.0.0.
```

Comment:
```suggestion
First, you need to install protobuf. The minimum Protobuf compiler (protoc) version required by ONNX is 3.0.0.
```
Shall we use Protobuf compiler (protoc) instead to prevent confusion with Python Protobuf version?